### ISSUE TITLE
rx 480 support

### ISSUE THREAD

**Issue by oldskoolau** [ZiftrCOIN/ziftrcoin#16]
any help on this would be great unable to mine at all

**Comment by ComputerCraftr** [comment]
This fork of sgminer was able to build a kernel and mine on my 4GB RX 480
http://cryptomining-blog.com/4563-new-sgminer-5-1-0-fork-with-ziftrcoin-support/

**Comment by oldskoolau** [comment]
yes it did generate a  ellesmere.bin  but runs a lower hash rate then my cpu i have tried adjusting the settings and is still weak im running rx 480s 8g cards  any sugestion on  settings  i also had a look at the bin compared to the other achritecures seems its not generating with the headers properly  i can only sumise that the drivers and CL versions dont support the rx card and are running on basic drivers being it was built 2 years before the card was relesed
